pancancer_classifier.py

```diff
@@ -248,8 +248,8 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
 sample_freeze = pd.read_csv(sample_freeze_file)
 
 # Construct data for classifier
-common_genes = set(mutation_df.columns.tolist()).intersection(genes)
-common_genes = list(common_genes.intersection(rnaseq_df.columns.tolist()))
+common_genes = set(mutation_df.columns).intersection(genes)
```

Comment:
why a set? 
Won't intersection() eliminate redundant terms? I might be wrong about that though